src/commands.js

```diff
@@ -43,15 +43,15 @@ function reportingClosure (action) {
     process.stdout.write(`${chalk.green(`[${currentlyMigrated}/${totalToMigrate}]`)} Successfully ${action} ${chalk.bold(migration.version)}: ${migration.description}\n`)
 }
 
-async function migrate ({ repoPath, migrations, to, dry, revertOk }) {
+async function migrate ({ repoPath, repoOptions, migrations, to, dry, revertOk }) {
```

Comment:
This file isn't used anymore since the cli.js is deleted, this can go away right?

Reply:
Good point, have removed.